content/docs/start/experiments.md

```diff
@@ -16,16 +16,49 @@ https://youtu.be/FHQq_zZz5ms
 
 In this section, we explore the basic features of DVC experiment management with
 the [`example-dvc-experiments`][ede] project.
-
 [ede]: https://github.com/iterative/example-dvc-experiments
 
+<details> 
```

Comment:
This change should get the app to build correctly 😃 

```suggestion
<details>
```

Reply:
I looked for the reason of failure for a few ~~hours~~  minutes and decided it's a local hiccup :)) Thank you @julieg18